x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_details.tsx

```diff
@@ -197,13 +225,27 @@ export const AlertDetails: React.FunctionComponent<AlertDetailsProps> = ({
             {uniqueActions && uniqueActions.length ? (
               <>
                 <EuiText size="s">
-                  <p>
-                    <FormattedMessage
-                      id="xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.actionsTex"
-                      defaultMessage="Actions"
+                  <FormattedMessage
+                    id="xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.actionsTex"
+                    defaultMessage="Actions"
+                  />{' '}
+                  {hasActionsWithBrokenConnector && (
+                    <EuiIconTip
+                      data-test-subj="actionWithBrokenConnector"
+                      type="alert"
+                      color="danger"
+                      content={i18n.translate(
+                        'xpack.triggersActionsUI.sections.alertsList.alertsListTable.columns.actionsWarningTooltip',
+                        {
+                          defaultMessage:
+                            'Unable to load a connector associated with this rule. Edit this rule to remove or select a new connector.',
```

Comment:
Pinging @elastic/kibana-docs to review this copy.

Reply:
Should we be suggesting to edit the rule or remove the connector as the action here?  What if it's simply an authentication error with the connector.   Wondering if we can get away with simply saying 'Unable to load a connector associated with this rule'.

Reply:
The mitigation step is to edit the rule to either (1) remove the connector or (2) select another connector for this action. At this stage, we don't know if there's any issue with the connector setup, just whether or not the connector exists.

I think if we do the warning callout as you suggested, we can simplify the tooltip to just say `Unable to load a connector associated with this rule`

Reply:
Makes sense! (Sorry I didn't respond to this comment earlier)